apps/mobile/src/screens/GroupWelcome/GroupWelcome.js

```diff
@@ -202,7 +203,7 @@ function LandingBodyContent ({ description, purpose, welcomePage }) {
     <>
       {welcomePage &&
         <View style={{ flexDirection: 'column', justifyContent: 'center' }}>
-          <HyloHTML html={welcomePage} />
+          <HyloHTML html={TextHelpers.markdown(welcomePage)} />
```

Comment:
we don't support markdown on the web right now. seems a little confusing to have both markdown and HTML support no? i mean i think its fine but just wanted to hear your reasoning.